pkg/plugin/pr-sanitizer/event_handler.go

```diff
@@ -82,25 +71,53 @@ func (gh *GitHubLabelsEventsHandler) HandleIssueCommentEvent(log log.Logger, com
 	return err
 }
 
-func (gh *GitHubLabelsEventsHandler) checkTitleAndSetStatus(log log.Logger, pullRequest *gogh.PullRequest) error {
-	change := ghservice.NewRepositoryChangeForPR(pullRequest)
-	statusContext := github.StatusContext{BotName: gh.BotName, PluginName: ProwPluginName}
-	statusService := status.NewStatusService(gh.Client, log, change, statusContext)
+func (gh *GitHubPRSanitizerEventsHandler) validatePullRequestTitleAndDescription(log log.Logger, pr *gogh.PullRequest) error {
+	change := ghservice.NewRepositoryChangeForPR(pr)
+	statusService := gh.newPRTitleDescriptionStatusService(log, change)
 
 	config := LoadConfiguration(log, change)
-	if gh.HasTitleWithValidType(config, *pullRequest.Title) {
-		return statusService.Success(SuccessMessage, SuccessDetailsPageName)
-	} else if prefix, ok := wip.GetWorkInProgressPrefix(*pullRequest.Title, wip.LoadConfiguration(log, change)); ok {
-		trimmedTitle := strings.TrimPrefix(*pullRequest.Title, prefix)
-		if gh.HasTitleWithValidType(config, trimmedTitle) {
-			return statusService.Success(SuccessMessage, SuccessDetailsPageName)
+	isTitleWithValidType := gh.HasTitleWithValidType(config, *pr.Title)
+	if !isTitleWithValidType {
+		if prefix, ok := wip.GetWorkInProgressPrefix(*pr.Title, wip.LoadConfiguration(log, change)); ok {
+			trimmedTitle := strings.TrimPrefix(*pr.Title, prefix)
+			isTitleWithValidType = gh.HasTitleWithValidType(config, trimmedTitle)
 		}
 	}
-	return statusService.Failure(TitleVerificationFailureMessage, TitleVerificationFailureDetailsPageName)
+
+	description, isIssueLinked := gh.GetDescriptionWithIssueLinkExcluded(pr.GetBody())
+
+	if len(description) < 50 {
+		message := fmt.Sprintf(DescriptionShortMessage, *pr.User.Login)
+		err := gh.Client.CreateIssueComment(scm.RepositoryIssue{
```

Comment:
We are just adding a comment here. I believe `StatusMessageService` is required more things than simply using `err := gh.Client.CreateIssueComment(*issue, &message)`

Reply:
`StatusMessageService` is adding a comment as well. It requires more things because it allows us modifying of the comment content as well as avoiding comment duplication. Anyway, that should be part of the #201